ocs_ci/ocs/ui/page_objects/s3_vector_tab.py

```
@@ -0,0 +1,375 @@
+            ("index_detail_distance_metric", "distance_metric", distance_metric),
+        ]
+
+        for locator_key, field, expected in detail_checks:
+            try:
+                value = self.get_index_detail_text(locator_key)
+                actual[field] = value
+                assert expected.lower() in value.lower(), (
+                    f"Index detail '{field}' mismatch: "
+                    f"expected '{expected}', got '{value}'"
+                )
+                logger.info(f"Verified {field}: '{value}' contains '{expected}'")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)